libs/sdk-model/src/dashboard/common.ts

```diff
@@ -1,9 +1,9 @@
 // (C) 2019-2022 GoodData Corporation
-import { ObjRef } from "@gooddata/sdk-model";
+import { ObjRef } from "../objRef";
 
 /**
  * Object unique identity
- * @public
+ * @alpha
```

Comment:
Another change from public to alpha, are we sure?

Reply:
Just revert of this: https://github.com/gooddata/gooddata-ui-sdk/commit/20469cf8480ad6439eb3942be7e8e8e72b3e1004#diff-39c41e614e5ee2582d18591ec1a85298dbbd607dab705950e78a75122ef83e08L6